src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -0,0 +1,403 @@
+# Electroencephalography (EEG)
+
+Support for EEG was developed as a BIDS Extension Proposal. Please cite the
+following paper when referring to this part of the standard in context of the
+academic literature:
+
+> Pernet, C. R., Appelhoff, S., Flandin, G., Phillips, C., Delorme, A., &
+> Oostenveld, R. (2018, December 6). BIDS-EEG: an extension to the Brain
+> Imaging Data Structure (BIDS) Specification for electroencephalography.
+> [https://doi.org/10.31234/osf.io/63a4y](https://doi.org/10.31234/osf.io/63a4y)
+
+The following EEG datasets have been formatted using the present specification
+and can be used for practical guidance when curating a new dataset.
+
+-   Single session per subject: [`eeg_matchingpennies`](https://doi.org/10.17605/OSF.IO/CJ2DR)
+-   Multiple sessions per subject: [`eeg_rishikesh`](https://doi.org/10.5281/zenodo.1490922)
+-   Combined with fMRI: [`eeg_rest_fmri`](https://osf.io/94c5t/files/)
+
+Further datasets are available from the [BIDS examples repository](https://github.com/bids-standard/bids-examples).
+
+## EEG recording data
+
+Template:
+
+```Text
+sub-<label>/
+    [ses-<label>]/
+      eeg/
+        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_eeg.<manufacturer_specific_extension>
+        sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>]_eeg.json
+```
+
+While there are many file formats to store EEG data, there are two officially
+supported data formats in EEG-BIDS: The [European data format](https://www.edfplus.info/)
```

Comment:
BIDS-EEG

Reply:
I would prefer avoiding the therm BIDS-EEG, BIDS-MEG, BIDS-iEEG in the spec. It creates a false impression there are multiple independent specs. Here, for example, it would suffice to just say 'BIDS'.

Reply:
+1, I changed to simply BIDS